packages/cache-core/src/cacheHandler.ts

```diff
@@ -132,6 +132,10 @@ export class Cache implements CacheHandler {
         return null
       }
 
+      if (data.value?.kind === 'ROUTE') {
+        return { ...data, value: { ...data.value, body: Buffer.from(data.value.body as unknown as string, 'base64') } }
```

Comment:
are we sure that for routes it returns buffer instead of json?

Reply:
Yeah, I tested it. It returns JSON which includes a Buffer body for api calls.